docs/quickstart.md

```diff
@@ -0,0 +1,176 @@
+# Initial Setup / Operations
+
+`SSO` is used to provide single-sign-on authentication and authorization for internal web
+applications, however, there are a few one-time prerequisites that need to be handled manually.
+
+**NOTE: all commands should be performed on your HOST OS.**
+
+## Google OAuth Setup
+
+Configuring `sso` requires that we first configure a Google OAuth Project.
```

Comment:
Here is where I might add a note that these steps (unfortunately) can't be automated.